deployments/fpga_plugin/intel-fpga-initcontainer.Dockerfile

```diff
@@ -68,8 +68,7 @@ RUN echo -e "{\n\
 
 # Setup
 
-RUN apk update
-RUN apk add rsync
+RUN swupd bundle-add network-basic
```

Comment:
what's the size impact with this change?

Reply:
There is a ~550MB difference compared with the previous one

Reply:
Image size increased from 20Mb to 570Mb, which is 2850% increase. Can we do it better?